lib/Backends/CPU/FunctionSpecializer.cpp

```diff
@@ -102,7 +104,15 @@ class FunctionSpecializer {
   /// \p arg. In the future, we may need to improve this logic by taking into
   /// account the semantics of the argument or even the specifics of the
   /// function call being specialized.
-  bool shouldSpecializeParameter(llvm::Value *arg) {
+  bool shouldSpecializeParameter(llvm::Value *arg, unsigned argIdx,
+                                 llvm::Function * /* unused */) {
```

Comment:
Let's give this parameter a name `callee`. But you can leave the comment that it is unused for now.